src/main/kotlin/com.bugsnag.android.gradle/BugsnagFileUploadTask.kt

```diff
@@ -8,6 +9,7 @@ interface BugsnagFileUploadTask {
     val endpoint: Property<String>
     val retryCount: Property<Int>
     val timeoutMillis: Property<Long>
+    val httpClientHelper: Property<BugsnagHttpClientHelper>
```

Comment:
This is a nice benefit of using the interface - we can provide this dynamically depending on gradle version in the plugin and tasks know nothing about its implementation